{{cookiecutter.repo_name}}/.github/workflows/main.yml

```diff
@@ -11,7 +11,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        python-version: [3.7, 3.8]
+        python-version: [3.6, 3.7, 3.8]
```

Comment:
Would you please also add 3.9 to the test matrix?

Reply:
Done in https://github.com/getpelican/cookiecutter-pelican-plugin/pull/12/commits/960948d4437760cfcaa91bdd700d0f3fe672a543